ChatExample/ChatExample/Screens/ChatExampleView.swift

```diff
@@ -24,6 +24,13 @@ struct ChatExampleView: View {
         ChatView(messages: viewModel.messages, chatType: .conversation) { draft in
             viewModel.send(draft: draft)
         }
+        .chatTheme(
```

Comment:
The background images have been included in the example, this will allow users to use chatBackgroundLight.jpg or chatBackgroundDark.jpg and allow them to see the image format that is needed for full screen background images